src/app/components/LanguageSwitcher.js

```diff
@@ -93,9 +93,7 @@ const LanguageSwitcher = (props) => {
                 </div>
                 { props.onSetDefault && languageCode !== primaryLanguage ?
                   <div>
-                    <IconButton onClick={handleClick}>
-                      <MoreVertIcon />
-                    </IconButton>
+                    <ButtonMain iconCenter={<MoreVertIcon />} variant="outlined" size="default" theme="text" onClick={handleClick} />
```

Comment:
Just out of curiosity, any reasoning behind explicitly passing size=default, since it's the default value for that prop?

Reply:
well only for 2 reasons really. 
1. for clarity sake when other devs look at the code there is less "hidden"
2. I was looking at other places where we might want to use the `ButtonMain` component that are standalone, meaning we might want to be able to set no size, no theme, etc and then style for that one location. Using css modules makes its harder if those styles are applied (ie. overriding parent styles from a child component). So there might be a case where there is no default prop set when empty, its just empty. 

That being said, I agree at the current state of this PR its largely unnecessary, but harmless in my opinion, just superfluous.